source/_components/octoprint.markdown

```diff
@@ -14,23 +14,99 @@ ha_release: 0.19
 ha_iot_class: "Local Polling"
 ---
 
-[OctoPrint](http://octoprint.org/) is a web interface for your 3D printer. This is the main component to integrate OctoPrint sensors, you will have to setup sensors and binary sensors separately.
+[OctoPrint](http://octoprint.org/) is a web interface for your 3D printer. This is the main component to integrate OctoPrint sensors.
 
-To get started with the OctoPrint API, please follow the directions on their [site](http://docs.octoprint.org/en/master/api/general.html). Once OctoPrint is configured you will need to add your API key and host to your `configuration.yaml`. 
+## {% linkable_title Base Configuration %}
 
+To get started with the OctoPrint API, please follow the directions on their [site](http://docs.octoprint.org/en/master/api/general.html). Once OctoPrint is configured you will need to add your API key and host to your `configuration.yaml`.
 
 ```yaml
 octoprint:
   host: YOUR_OCTOPRINT_HOST
   api_key: YOUR_API_KEY
-  bed: false
-  number_of_tools: 1
 ```
 
-Configuration variables:
+{% configuration %}
+host:
+  description: IP address or hostname of Octoprint host.
+  required: true
+  type: string
+api_key:
+  description: The retrieved api key.
+  required: true
+  type: string
+name:
+  description: The name for this printer, must be unique if multiple printers are defined.
+  required: false
+  type: string
+  default: 'OctoPrint'
+port:
+  description: The port of the Octoprint server.
+  required: false
+  type: int
```

Comment:
`int` -> `integer`

Reply:
I will change that but FYI the [documentation](https://developers.home-assistant.io/docs/en/documentation_create_page.html) says: 

> type:: The type of the variable. Allowed entries: string, int, time, template or map

It should probably be updated :)